pkg/consts/konnect.go

```diff
@@ -0,0 +1,11 @@
+package consts
+
+const (
+	// CleanupPluginBindingFinalizer is the finalizer attached to entities that are properly
+	// referenced by KongPluginBindings, that should be cleaned up when the KongService
+	// gets deleted.
```

Comment:
1. I believe this comment should be a bit more explicit. What does "properly referenced" mean in here?
2. "that should be cleaned up when the KongService gets deleted":
  - I believe we can use a generic term like `resource` instead of KongService to prevent us forgetting to update this comment when other types of objects are supported.
  - What should clean this finalizer? I believe this is worth to be mentioned, otherwise users might think that they are somehow expected to clean this up.

Reply:
yep, I agree. here it is c4a737641026a56f76d2fc0d75a84d0c41187ce8